app/services/api/v3/ecf/partnerships_query.rb

```diff
@@ -20,7 +20,15 @@ def partnerships
             .where(partnerships: { cohort: cohorts })
             .order(sort_order(default: "partnerships.created_at ASC", model: Partnership))
             .distinct
-          scope = scope.where("partnerships.updated_at > ?", updated_since) if updated_since_filter.present?
+
+          if updated_since_filter.present?
+            scope = scope.includes(:delivery_partner, school: [:induction_coordinator_profiles])
+            scope = scope.where(partnerships: { updated_at: updated_since.. })
+              .or(scope.where(school: { updated_at: updated_since.. }))
+              .or(scope.where(delivery_partner: { updated_at: updated_since.. }))
+              .or(scope.where(school: { induction_coordinator_profiles: { updated_at: updated_since.. } }))
```

Comment:
```suggestion
              .or(scope.where(school: { users: { updated_at: updated_since.. }}))
```
Induction coordinator names/emails (users) are what are changing so we have to surface those values :)